interop/jvm/src/test/scala/scalaz/zio/interop/futureSpec.scala

```diff
@@ -101,4 +103,32 @@ class futureSpec(implicit ee: ExecutionEnv) extends AbstractRTSSpec {
     unsafeRun(failedIO.toFutureE(new Exception(_))) must throwA[Exception](message = "IOs also can fail").await
   }
 
+  val lazyOnParamRefFiber = {
+    var evaluated = false
+    def ftr       = Future { evaluated = true }
+    Fiber.fromFuture(ftr)(ec)
+    evaluated must beFalse
+  }
+
+  val lazyOnParamInlineFiber = {
+    var evaluated = false
+    Fiber.fromFuture(Future { evaluated = true })(ec)
+    evaluated must beFalse
+  }
+
+  val catchBlockExceptionFiber = {
+    def noFuture: Future[Unit] = throw new Exception("no future for you!")
+    unsafeRun(Fiber.fromFuture(noFuture)(ec).join) must throwA[Exception](message = "no future for you!")
```

Comment:
Sorry for the post-merge nitpick, but wouldn't this test be better if you split the creation of the Fiber and the running of it into separate lines? If I understand correctly, `Fiber.fromFuture(...)` should not throw the exception, but `unsafeRun` should.

Reply:
That sounds reasonable to me :)